infra/Pos/Util/TimeWarp.hs

```diff
@@ -42,3 +46,7 @@ nodeIdToAddress (NodeId ep) = do
     (hostName, strPort, _) <- TCP.decodeEndPointAddress ep
     port <- readMaybe strPort
     return (BS8.pack hostName, port)
+
+-- | Parsed for network address in format @host:port@.
+addrParser :: P.Parser NetworkAddress
+addrParser = (,) <$> (encodeUtf8 <$> P.host) <*> (P.char ':' *> P.port)
```

Comment:
Looks much like [`recipient` parser](https://github.com/serokell/serokell-util/blob/867ec23923869aed3ba3367e30ad435b60721060/src/Serokell/Util/Parse/Network.hs#L125) from `serokell-util`. I'm for removing this duplication somehow. It will also remove `parsec` dependency, I guess.